src/components/children/NestedInfoTable.jsx

```diff
@@ -167,7 +167,7 @@ class NestedInfoTable extends React.Component {
 								})
 							) : key ==='dataType' ? // Another ternary operator to check if the value is a dataType if so create linking
 									(<TableCell>
-										<a href={`${config.hostname}/semantics/api/types/${data[key]}`} target="_blank" rel="noopener noreferrer">{data[key]} </a>
+										<a href={`/semantics/api/types/${data[key]}`} target="_blank" rel="noopener noreferrer">{data[key]} </a>
```

Comment:
1. can you put the `/semantics/api/types/` part in config?
2. Could you also restrict this to only incore: or ergo: ones? maybe use some regex?